### DISCUSSION TITLE
有什么什么办法能解决那个什么qq版本过低的问题啊？我真的快被逼疯了

### DISCUSSION THREAD

**Discussion by SolanumNigrum** [simple-robot/Discussions#56] [Q&A]
一启动就报那个什么qq版本过低的问题，根本不知道怎么解决（更新qq了没用），连启动都启动不了根本没办法往下一步走啊。有没有大佬告诉我怎么解决？

**Comment by ForteScarlet** [comment]
参考
- https://github.com/simple-robot/Discussions/discussions/20
- https://github.com/simple-robot/Discussions/discussions/55

及其他社区与issues中相关问答
